source/dom.js

```diff
@@ -1,16 +1,14 @@
-'use strict';
+const div = document.createElement('div');
+let gid = 0;
 
-var div = document.createElement('div');
-var gid = 0;
-
-var dom = module.exports = {
+const dom = {
```

Comment:
This (`dom.js`) and several other (`history.js`, `utils.js`) files export an object, but are actually just a set of not-so-tightly-coupled functions. I believe they can be now exported directly (`export const swapSVG = () => {}`) instead of wrapping them into an object.